packages/@lightningjs/ui-components/src/components/Tooltip/Tooltip.stories.js

```diff
@@ -58,24 +58,41 @@ export default {
   }
 };
 
-export const Basic = () =>
+const sharedArgActions = {
```

Comment:
Can we change this to `sharedParameters` since the argActions are nested inside the object?

Reply:
Yes, I can change to sharedParameters, but crosschecked with other components, they are all using sharedArgActions? Either way is fine for me.

Reply:
I think it's fine to keep it as arg actions to be consistent for now and we can always make a PR to update across the board later